backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,422 @@
+
+Some games use separate audio *and* video streams (files). Even if the CPU is
+able to handle both, the bottleneck becomes ... disk access. This is visible in
+The Curse Of Monkey Island for example -- there's audible stuttering during the
+intro sequence (and during the game as well). Increasing "output_samples" makes
+the rendering literally crawling! Why? Because disk I/O is busy with loading
+even *more* sample data so there's less time for video loading and rendering.
+Try to put "musdisk1.bun" and "musdisk2.bun" into a ramdisk (i.e. u:/ram in
+FreeMiNT), you'll be pleasantly surprised with the performance boost gained.
+
+"Mute" vs. "Mute all" in GUI vs. "No music" in GUI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Not the same thing. "Mute" (available only via the shortcut CONTROL+u) generates
+an event to which the sample mixer can react (i.e. stop mixing the silence...).
+
+"Mute all" doesn't generate anything, it basically just lowers the volume of the
+music to zero.
+
+"No music" means using the null audio plugin which prevents generating any MIDI
+music (and therefore avoiding the expensive synthesis emulation) but beware, it
+doesn't affect CD (*.wav) playback at all!
+
+So for the best performance, always choose "No music" in the GUI options when
+the game contains MIDI tracks and "Mute" when the game contains a sampled
+soundtrack.
+
+Please note that it is not that bad, you surely can play The Secret of Monkey
+Island with AdLib enabled (but the CD/talkie versions sound better and
+are cheaper to play ;)).
+
+Vsync in GUI
+~~~~~~~~~~~~
+
+Carefully with the vsync option. It can easily cripple direct/single buffer
+rendering by 10-15 FPS if not used with caution. That happens if a game takes,
+say, 1.2 frames per update (so causing screen tearing anyway and rendering the
+option useless) but Vsync() forces it to wait 2 full frames instead.
+
+By the way, the vsync flag in Global Options affects also the overlay rendering
+(with all the pitfalls which apply to the single buffering mode)
+
+Slow GUI
+~~~~~~~~
+
+Themes handling is quite slow - each theme must be depacked, each one contains
+quite a few XML files to parse and quite a few images to load/convert. Best is
+to use the built-in one. A compromise solution is to depack the theme in an
+equally named directory. It will be loaded, too but without the depacking
+overhead.
+
+
+Known issues
+------------
+
+- aspect ratio correction works on RGB only (yet)
+
+- SuperVidel's DVI output is stretched when in 320x200 or 640x400; I'll  wait
+  for other people's experiences, maybe only my LCD is so lame.
+
+- adding a game in TOS and loading it in FreeMiNT (and vice versa) generates
+  incompatible paths. Either use only one system or edit scummvm.ini and set
+  there only relative paths (mintlib bug/limitation).
+
+- the default theme ("ScummVM Modern Theme Remastered") is not loaded
+  automatically on plain 8+3 filesystem. If you want to use it nevertheless,
```

Comment:
Shall we then rename the theme files?

Reply:
I didn't dare to ask for something like that but if you could, that would be a tremendous help because I can't easily work around it (our file system is able to do "some long name.zip" to "some_lon.zip" automatically however ScummVM is too clever for this: it first lists the content of a directory and then checks for concrete names, i.e. I get "some_lon.zip" as found but ScummVM asks "have we read some long name.zip previously?").

Reply:
To clarify, is the problem with the theme .zip names, not with the files that are inside of those zips?

Reply:
Indeed. However on a second thought and in the context of your [other comment](https://github.com/scummvm/scummvm/pull/4687/files#r1112394997) I have decided to take inspiration from other ports, i.e. override "gui_theme" from my backend, namely to the built-in one:

- If we were to change "scummvmremastered.zip" to, say, "remaster.zip", we'd need to change this name also in gui-manager.cpp and even let a comment there that this name must always conform to 8+3 format. I think this is too restricting.
- I have done some measurements on a slow CPU to better see the difference:
  - loading remaster.zip (just renamed): 1m52s
  - loading remaster.zip (repacked with `-0`): 1m45s
  - falling back to the built-in one: 0m50s

No further work is needed here then.

Reply:
Ah, that's nice. However, a little work is still needed :D

In the`ports.mk`, where there is an install target, perhaps, you need to implement this renaming for your platform, especially if/when somebody wants to cross-build it on an FS with long filenames support.

Reply:
On the contrary -- choosing the built-in theme as default gets rid of this problem entirely. If someone wants to use a slower but prettier theme, all that is needed is go to the options and let ScummVM list all the zip files, no matter how much screwed they are. ScummVM will find scummrem.zip for instance, and this will be stored in `scummvm.ini`. So no renaming is needed anymore. Win-win!